src/parser/mod.rs

```diff
@@ -13475,13 +13477,23 @@ impl<'a> Parser<'a> {
 }
 
 impl Word {
+    #[deprecated(since = "0.54.0", note = "please use `into_ident` instead")]
```

Comment:
I am trying to help any downstream crates that may also use this function with a deprecation notice rather than simply removing the function

Reply:
Sounds good!